spec/models/oseq_flowcell_spec.rb

```diff
@@ -35,7 +35,11 @@
       tag2_identifier: '2',
       tag2_sequence: 'GTCA',
       tag2_set_id_lims: 'second_tag_set',
-      tag2_set_name: 'tag_set_2'
+      tag2_set_name: 'tag_set_2',
+      flowcell_id: 'FAV1234',
```

Comment:
The flowcell ID is the key one for us, thanks.

A "nice to have" would be a column for the "official" run identifier that is generated by ONT's software. However, that could complicate ownership of the oseq_flowcell table because NPG would be best placed to populate that value into the column, rather than yourselves. The run identifier is not generated until the run has started, so we're the first to see it as we collect the sequencing run data from the instrument. I mention is now because we're talking about adding columns, but more FYI than anything.

Reply:
> The flowcell ID is the key one for us, thanks.
> 
> A "nice to have" would be a column for the "official" run identifier that is generated by ONT's software. However, that could complicate ownership of the oseq_flowcell table because NPG would be best placed to populate that value into the column, rather than yourselves. The run identifier is not generated until the run has started, so we're the first to see it as we collect the sequencing run data from the instrument. I mention is now because we're talking about adding columns, but more FYI than anything.

I am happy to add the column. What would you like to call it? And what type would it be? We had a brief discussion about how to populate it. As it is a PSD owned table we would prefer it to be populated via a message. More than happy to discuss this further. For now if you let me know. The name of the column, description for the column, column type and any other constraints.

Reply:
Thanks, sounds good.

Name: `run_id`
Type: `varchar`
Constraints: Unique, nullable
Description: Run identifier assigned by MinKNOW

All cases I've looked at so far have been 30 character strings.